File: documentation/user/en/solve/examples/render-category-menu/hybrid-menu.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		HierarchyWithin(
        			"categories",
        			AttributeEquals("code", "over-ear")
        		)
        	),
        	Require(
        		HierarchyOfReference(
        			"categories",
        			RemoveEmpty,
        			FromRoot(
        				"topLevel",
        				EntityFetch(
        					AttributeContent("code")
        				),
        				StopAt(
        					Level(1)
        				)
        			),
        			Siblings(
        				"siblings",
        				EntityFetch(
        					AttributeContent("code")
        				),
        				StopAt(
        					Distance(1)
        				)
        			),
        			Parents(
        				"parents",
        				EntityFetch(
        					AttributeContent("code")
        				)
        			)
        		)
        	)
        )
	)
);

File: documentation/user/en/solve/examples/render-category-menu/hybrid-menu.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					hierarchyWithin(
						"categories",
						attributeEquals("code", "over-ear")
					)
				),
				require(
					hierarchyOfReference(
						"categories",
						fromRoot(
							"topLevel",
							entityFetch(
								attributeContent("code")
							),
							stopAt(
								level(1)
							)
						),
						siblings(
							"siblings",
							entityFetch(
								attributeContent("code")
							),
							stopAt(
								distance(1)
							)
						),
						parents(
							"parents",
							entityFetch(
								attributeContent("code")
							)
						)
					)
				)
			)
		);
	}
);
